### PR TITLE
WIP - Ensure scalers are only listed for a given version

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Let's see if this works since I cannot fully run it locally.

**Comment by tomkerkhove** [comment]
Had a working version, cleaned things up and messed it up again 😂

The fun of frontend...

**Comment by lucperkins** [comment]
I think the issue is that you’re using \\ as the file path separator instead of / further down

File: layouts/partials/scalers.html [hunk 1]
```diff
@@ -1,10 +1,15 @@
 {{ $here    := $.Page.RelPermalink }}
 {{ $scalers := where site.RegularPages ".CurrentSection.Title" "Scalers" }}
+{{ $thisVersion    := index (split .File.Path "/") 1 }}
 <div class="buttons has-extra-top-margin">
   {{ range $scalers }}
   {{ $isHere := eq .RelPermalink $here }}
+  {{ $version := index (split .File.Path "\\") 1 }}
```
Comment (lucperkins) [inline layouts/partials/scalers.html:7]: Use the `/` separator here as well, instead of `\\`

**Comment by lucperkins** [comment]
@tomkerkhove What's the issue when you run it locally?

**Comment by zroubalik** [comment]
@tomkerkhove have you been able to fix this one?

**Comment by tomkerkhove** [comment]
Nope, unfortunately not...

File: layouts/partials/scalers.html (hunk 1 above)
Comment (tomkerkhove) [reply layouts/partials/scalers.html]: This works fine on my machine:
`{{ $version := index (split .Dir "\\") 1 }}`

But must be because I'm on Windows.

**Comment by tomkerkhove** [comment]
The current approach works on my box (Windows) by using `\\` but if I move it to `/`. Any pointers for this @lucperkins ?

Local:
image

**Comment by lucperkins** [comment]
I don't think there's a way around this problem. The only way to infer the version of a given document is to read it from the filepath. Since the production site is built on Linux, you'll need to use `/`.

**Comment by tomkerkhove** [comment]
That's what I do indeed, but the build is failing then and I'm not sure why/how to fix it.

**Comment by tomkerkhove** [comment]
Closing in favor of #200

**Pull request closed without merging**